app/ui/src/app/integration/edit-page/flow-view/flow-view.component.html

```diff
@@ -106,10 +110,12 @@
       </div>
     </div>
   </syndesis-loading>
-  <button class="btn btn-default toggle-collapsed"
+  <button
+    class="btn btn-default toggle-collapsed"
     (click)="toggleCollapsed()"
     [class.collapsed]="isCollapsed"
     [tooltip]="toggleButtonTooltip"
+    aria-label="Integration Visualization Pane Details Toggle"
```

Comment:
"Toggle" should probably be the first word in the sentence. And maybe it could just read "Toggle integration details"?